web/client/components/mapcontrols/measure/MeasureResults.jsx

```diff
@@ -0,0 +1,181 @@
+ */
+
+const React = require('react');
+const {Panel} = require('react-bootstrap');
+const Draggable = require('react-draggable');
+
+const Message = require('../../../plugins/locale/Message');
+const ReactIntl = require('react-intl');
+const FormattedNumber = ReactIntl.FormattedNumber;
+
+const {isEqual} = require('lodash');
+
+const MeasureResults = React.createClass({
+    propTypes: {
+        id: React.PropTypes.string,
+        name: React.PropTypes.string,
+        columnProperties: React.PropTypes.object,
+        propertiesChangeHandler: React.PropTypes.func,
+        lengthLabel: React.PropTypes.oneOfType([React.PropTypes.string, React.PropTypes.element]),
+        areaLabel: React.PropTypes.oneOfType([React.PropTypes.string, React.PropTypes.element]),
+        bearingLabel: React.PropTypes.oneOfType([React.PropTypes.string, React.PropTypes.element]),
+        uom: React.PropTypes.shape({
+                    length: React.PropTypes.shape({ unit: React.PropTypes.string.isRequired,
+                              label: React.PropTypes.string.isRequired}),
+                    area: React.PropTypes.shape({ unit: React.PropTypes.string.isRequired,
+                            label: React.PropTypes.string.isRequired})
+                        }),
+        toggleMeasure: React.PropTypes.func,
+        measurement: React.PropTypes.object,
+        lineMeasureEnabled: React.PropTypes.bool,
+        areaMeasureEnabled: React.PropTypes.bool,
+        bearingMeasureEnabled: React.PropTypes.bool,
+        separatePanel: React.PropTypes.bool,
+        title: React.PropTypes.object,
+        panelStyle: React.PropTypes.object,
+        panelClassName: React.PropTypes.string
+    },
+    getDefaultProps() {
+        return {
+            columnProperties: {
+                xs: 4,
+                sm: 4,
+                md: 4
+            },
+            id: "measure-result-panel",
+            uom: {
+                length: {unit: 'm', label: 'm'},
+                area: {unit: 'sqm', label: 'm²'}
+            },
+            separatePanel: true,
+            title: <Message msgId="measureComponent.title"/>,
+            panelStyle: {
+                minWidth: "300px",
+                left: "52px",
+                zIndex: 100,
+                position: "absolute",
+                overflow: "auto"
+            },
+            panelClassName: "drawer-menu-panel"
+        };
+    },
+    shouldComponentUpdate(nextProps) {
+        return !isEqual(nextProps, this.props);
+    },
+    onModalHiding() {
+        const newMeasureState = {
+            lineMeasureEnabled: false,
+            areaMeasureEnabled: false,
+            bearingMeasureEnabled: false,
+            geomType: null,
+            // reset old measurements
+            len: 0,
+            area: 0,
+            bearing: 0
+        };
+        this.props.toggleMeasure(newMeasureState);
+    },
+    getFormattedBearingValue(azimuth = 0) {
```

Comment:
Can we move common functions to a utility module and use them in both MeasureComponent and MeasureResults?